index.bs

```diff
@@ -2743,7 +2743,7 @@ PDF document represented as a Base64-encoded string.
         ? margin: browsingContext.PrintMarginParameters,
         ? orientation: ("portrait" / "landscape") .default "portrait",
         ? page: browsingContext.PrintPageParameters,
-        ? pageRanges: [*(js-uint / text)],
+        ? pageRanges: [*(js-uint / text .regexp "^[0-9]+(?:-[0-9]+)?$" )],
```

Comment:
The algorithm (https://w3c.github.io/webdriver/#dfn-parse-a-page-range) is responsible to validate this.

Adding a regexp here is "premature optimization".

I don't think it hurts, and it happens to play well with our zod generator, but see what the Mozilla folks think about it.

PS. This matches "1-3", but it does not match "1-" nor "-3", however AFAIK it should (please double check).

Reply:
Now it currently matches "-". Is it allowed?

Reply:
Yes, it is allowed.